### PR TITLE
A3 MapLoader, Map, & Game classes full refactor complete!

### PR DESCRIPTION
**Pull request by RemineralizedWater**
A3 MapLoader, Map, & Game classes full refactor complete!

### CODE REVIEW

**Comment by RemineralizedWater** [comment]
A3 MapLoader, Map, & Game classes full refactor complete!

File: src/A2_Part6_Driver.cpp [hunk 1]
```diff
@@ -6,16 +6,16 @@
 #include "Game/Game.h"
 
 /**
- *  //count the player with most elixers win 2 victory points.if tied then each receive 1
+ *  //count the player with most elixirs win 2 victory points.if Tied then each receive 1
```
Comment (whuras) [inline src/A2_Part6_Driver.cpp]: haha did you just do a find and replace for the word exilers and tied?

File: src/Game/Game.cpp [hunk 2]
```diff
@@ -88,197 +93,198 @@ vector<Player*> Game::createPlayers(int startingPoint) {
...
     }
-    players[winner]->SetVictoryPoints(players[winner]->GetVictoryPoints()+1);
+    players[winner]->SetVictoryPoints(players[winner]->GetVictoryPoints() + 1);
     return false;
 }
 
-
-Map * Game::selectMap(bool &validMap) {
-
+Map *Game::selectMap(bool &validMap) {
```
Comment (whuras) [inline src/Game/Game.cpp]: Should be PascalCase

File: src/A2_Part6_Driver.cpp (hunk 1 above)
Comment (RemineralizedWater) [reply src/A2_Part6_Driver.cpp]: lol yea tied it shouldn't have changed the comments. There's a method I changed from tied to Tied and it did that. Elixir I fixed spelling (so irrelevant, just me being thorough).

**Comment by RemineralizedWater** [comment]
Okay pushed the fixes requested!

**Review by whuras — APPROVED** [review]

**Pull request merged**